test/common/testlib.py

```diff
@@ -1604,7 +1604,7 @@ def terminate_sessions():
 
             # Restart logind to mop up empty "closing" sessions, and clean user id cache for non-system users
             self.machine.execute("systemctl stop systemd-logind; cd /run/systemd/users/; "
-                                 "for f in *; do [ $f -le 500 ] || rm $f; done")
+                                 "for f in $(ls); do [ $f -le 500 ] || rm $f; done")
```

Comment:
FTR: It would be more efficient to do `[ -e $f -a $f -le 500 ]`, as that avoids spawning a shell and `ls`.